maim.1

```diff
@@ -40,7 +40,7 @@ Sets the time in seconds to wait before taking a screenshot. Prints a simple mes
 By default maim super-imposes the cursor onto the image, you can disable that behavior with this flag.
 .TP
 .BR \-m ", " \-\-quality
-An integer from 1 to 10 that determines the compression quality. 1 is the highest (and lossiest) compression available for the provided format. For example a setting of `1` with png (a lossless format) would increase filesize and decrease decoding time. While a setting of `1` on a jpeg would create a pixel mush. No effect on bmp images.
+An integer from 1 to 10 that determines the compression quality. 1 is the highest (and lossiest) compression available for the provided format. For example a setting of `1` with png (a lossless format) would increase filesize and decrease decoding time. While a setting of `1` on a jpeg would create a pixel mush. No effect on bmp images. 0-9 set the quality of lossy webp, while 10 will encode as lossless.
```

Comment:
"An integer from 1 to 10". I guess this whole line in manual should be rewritten to not be very misleading?

Reply:
I don't know about the "An integer from 1 to 10" that you mentioned, but I agree that the rest of the line is weird, especially with the special case of png, where the correlation with file size is different. I'll try to rewrite that to be more clear.

Reply:
Yeah I'm not sure what I was thinking when I wrote that. I probably got confused because both the jpg library and the PNG one refer to the setting as "quality". PNG being backwards in use didn't really raise any alarms to me in 2016 haha. I'm too afraid to fix it since a lot of people probably rely on it being wrong now.

Reply:
I believe it should be rewritten from a single line to a list.
I see it like this:

```
--quality
For bmp: no effect
For jpeg: 1-10, higher is better quality and bigger file, always lossy
For png: 1-10, higher is better compression, always lossless
For webp: 0-10, higher is better quality and bigger file, lossy for values in range 0-9, lossless for 10
```

Reply:
> I don't know about the "An integer from 1 to 10" that you mentioned

I don't like how manual says first that the value should be in range from 1 to 10, but suddenly a value of 0 appears for webp in the end.